js/spaceShip.js

```diff
@@ -1,33 +1,62 @@
-var SpaceShip = function(x, y){
+
+
+  //default parameters are to put the spaceShip in the center
+var SpaceShip = function (x = width / 2, y = height / 2) {
   //Variables here
   this.pos = createVector(x, y); //Vector for holding position
-  this.acc = createVector(0, 0); //No acceleration by default
+  this.vel = createVector(0, 0); // Vel changes position
+  this.img = loadImage("images/spaceship.png");
+  let acc = createVector(0,0);
 
   //Functions here
-  this.update = function(){
-    this.pos.add(this.acc);
+  this.update = function update() {
+    this.vel.add(acc)
+    this.pos.add(this.vel);
+    acc.set(0,0);
   }
 
-  this.move = function(moveCode){
-    switch(moveCode){
+  function applyForce(force){
+    acc.add(force);
+  }
+
+  this.move = function (moveCode) {
+    let forceVector = createVector(0,0);
+    switch (moveCode) {
       //TODO make decisions here and apply Force towards that side in acc
-      // this.acc.add(forceVector);
+      // acc.add(forceVector);
+      case DOWN_ARROW:
+        forceVector.set(0,1);
+        break;
+      case UP_ARROW:
+        forceVector.set(0,-1);        
+        break;
+      case LEFT_ARROW:
+        forceVector.set(-1,0);
+        break;
+      case RIGHT_ARROW:
+        forceVector.set(1,0);
+        break;
     }
+    applyForce(forceVector);
   }
 
+ 
+
   //Generates a bullet with a force,  upwards
-  this.shoot = function(){
-    // TODO
+  this.shoot = function () {
+    var bullet = new Bullet(this.pos.x, this.pos.y, true);
   }
 
   //
-  this.draw = function(){
-    // TODO
+  this.draw = function () {
+    image(this.img, this.pos.x, this.pos.y, 100, 65);
   }
 }
 
 
 //Generate SpaceShip with constructor, give the x and y
-function placeSpaceShip(){
+function placeSpaceShip() {
+  const spaceShip = new SpaceShip(canvas.width / 2, canvas.height - canvas.height / 5);
```

Comment:
Use var please :P
Also push spaceship to gameObjects so its update and draw functions can be called